### PR TITLE
Remove rendezvous point default

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

**Comment by da-kami** [comment]
Looks good - I will adapt the `discover-and-take` script to add the rendezvous-node parameter and push a commit here.

**Comment by da-kami** [comment]
I will adapt the READMEs now (at least the one for the CLI needs a small update concerning the default output of the rendezvous-point. 
OK if I mention our rendezvous point in the readme?

**Review by da-kami — APPROVED** [review]

File: docs/cli/README.md [hunk 1]
```diff
@@ -94,13 +94,13 @@ FLAGS:
     -V, --version    Prints version information
 
 OPTIONS:
-        --rendezvous-point <rendezvous-point>       Address of the rendezvous point you want to use to discover ASBs [default: /dnsaddr/rendezvous.coblox.tech/p2p/12D3KooWQUt9DkNZxEn2R5ymJzWj15MpG6mTW84kyd8vDaRZi46o]
+        --rendezvous-point <rendezvous-point>       Address of the rendezvous point you want to use to discover ASBs
         --tor-socks5-port <tor-socks5-port>         Your local Tor socks5 proxy port [default: 9050]
 ```
 
 This command only takes optional parameters and can be run as-is:
```
Comment (thomaseizinger) [inline docs/cli/README.md:101]: This is no longer true :D

```suggestion
```

Comment (da-kami) [reply docs/cli/README.md:101]: Oh, that one is in there twice? 😅

**Pull request merged**